### PR TITLE
8198619: java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java fails on mac

### PR DESCRIPTION
**Pull request by prsadhuk**
This test was failing in CI nightly testing due to samevm mode issue. 
Modified test to use waitForIdle() judiciously, dispose frame in finally block and move the frame to center of screen.
Several iterations of the test pass in all platforms. Link in JBS.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8198619](https://bugs.openjdk.java.net/browse/JDK-8198619): java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java fails on mac

### Reviewers
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3618/head:pull/3618` \
`$ git checkout pull/3618`

Update a local copy of the PR: \
`$ git checkout pull/3618` \
`$ git pull https://git.openjdk.java.net/jdk pull/3618/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3618`

View PR using the GUI difftool: \
`$ git pr show -t 3618`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3618.diff">https://git.openjdk.java.net/jdk/pull/3618.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
How did you check that the problem was in the samevm mode?

**Comment by prsadhuk** [comment]
I did not have the logs but it was raised around the same time of other samevm issue Feb 2018 ie, the JBS tests mentioned in JDK-8254976

File: test/jdk/java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java [hunk 1]
```diff
@@ -58,71 +58,77 @@
...
-        }
+            for (int i = 0; i < nx * ny / 2; i++) {
+                robot.keyPress(KeyEvent.VK_TAB);
+                robot.keyRelease(KeyEvent.VK_TAB);
+                robot.waitForIdle();
+            }
 
-        robot.waitForIdle();
-        robot.delay(200);
+            robot.waitForIdle();
```
Comment (mrserb) [inline test/jdk/java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java:81]: Looks like this is not needed, a line above we call it already(in the loop.)

Comment (prsadhuk) [reply test/jdk/java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java]: Removed..

**Comment by prsadhuk** [comment]
Any further comment on this?

**Review by aivanov-jdk — APPROVED** [review]

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**